.github/workflows/rws-submissions-checks.yml

```diff
@@ -46,20 +46,25 @@ jobs:
           CONTENTS: ${{ steps.read_results.outputs.contents }}
         run: echo "$CONTENTS"
       - name: Create Comment
-        if: steps.read_results.outputs.contents != 'success'
+        if: steps.read_results.outputs.contents != 'success' && ${{ !startsWith(steps.read_results.outputs.contents, 'Warning')}}
```

Comment:
Does this support emitting a warning alongside errors? What if the warning happens to be the first line that was emitted, but it's followed by errors?

(Maybe we ought to write separate files for warnings/errors.)

Reply:
Do you want help me?

Reply:
> Does this support emitting a warning alongside errors? What if the warning happens to be the first line that was emitted, but it's followed by errors?
> 
> (Maybe we ought to write separate files for warnings/errors.)